demisto_sdk/commands/lint/tests/test_pylint_plugin/base_checker_test.py

```diff
@@ -343,8 +343,11 @@ def test_regular_if_else_checker(self):
                     msg_id='unimplemented-commands-exist',
                     node=node_a,
                     args=str(['test2']),
-
                 ),
+                pylint.testutils.Message(
```

Comment:
I think this is not needed, as this message is not relevant to this test.
It might be better to add the self.checker.is_script = True in the beginning to avoid unnecessary checks inside a specific behaviour test